int16 x
int16 y
---
nav_msgs/OccupancyGrid map
